linux/process.go

```diff
@@ -11,15 +11,20 @@ import (
 	"github.com/containerd/containerd/runtime"
 )
 
+// Process implements a linux process
 type Process struct {
 	id string
 	t  *Task
 }
 
+// ID of the process
```

Comment:
`ID returns the process ID` ?

Reply:
no, this is not how you phase it, you want to not have redundant information like repeating the name of the method in the comment